diff --git a/node_modules/@poppinss/utils/build/index.js b/node_modules/@poppinss/utils/build/index.js
index 84ca468..10ba893 100644
--- a/node_modules/@poppinss/utils/build/index.js
+++ b/node_modules/@poppinss/utils/build/index.js
@@ -111,19 +111,6 @@ var RuntimeException = class extends Exception {
   static status = 500;
 };
 
-// src/import_default.ts
-async function importDefault(importFn, filePath) {
-  const moduleExports = await importFn();
-  if (!("default" in moduleExports)) {
-    const errorMessage = filePath ? `Missing "export default" in module "${filePath}"` : `Missing "export default" from lazy import "${importFn}"`;
-    throw new RuntimeException(errorMessage, {
-      cause: {
-        source: importFn
-      }
-    });
-  }
-  return moduleExports.default;
-}
 
 // src/define_static_property.ts
 import lodash from "@poppinss/utils/lodash";
@@ -242,30 +229,6 @@ function isScriptFile(filePath) {
   return false;
 }
 
-// src/fs_import_all.ts
-async function importFile(basePath, fileURL, values, options) {
-  const filePath = fileURLToPath2(fileURL);
-  const fileExtension = extname2(filePath);
-  const collectionKey = relative(basePath, filePath).replace(new RegExp(`${fileExtension}$`), "").split(sep);
-  const exportedValue = fileExtension === ".json" ? await import(fileURL, { assert: { type: "json" } }) : await import(fileURL);
-  lodash2.set(
-    values,
-    options.transformKeys ? options.transformKeys(collectionKey) : collectionKey,
-    exportedValue.default ? exportedValue.default : { ...exportedValue }
-  );
-}
-async function fsImportAll(location, options) {
-  options = options || {};
-  const collection = {};
-  const normalizedLocation = typeof location === "string" ? location : fileURLToPath2(location);
-  const files = await fsReadAll(normalizedLocation, {
-    filter: isScriptFile,
-    ...options,
-    pathType: "url"
-  });
-  await Promise.all(files.map((file) => importFile(normalizedLocation, file, collection, options)));
-  return collection;
-}
 
 // src/message_builder.ts
 var MessageBuilder = class {
@@ -406,11 +369,9 @@ export {
   createError,
   defineStaticProperty,
   flatten,
-  fsImportAll,
   fsReadAll,
   getDirname,
   getFilename,
-  importDefault,
   isScriptFile,
   joinToURL,
   naturalSort,
