diff --git a/dist/index.cjs b/dist/index.cjs
index 8623625b851577972579c3a1388578c7732527b3..7cb7833b43341f8aa65c6e922528ccf4b2d8412a 100644
--- a/dist/index.cjs
+++ b/dist/index.cjs
@@ -1,6 +1,6 @@
 'use strict';
 
-const ts = require('typescript');
+const ts = require('typescript-legacy');
 const core = require('./core.cjs');
 const fallback = require('./shared/twoslash.D2-zMJku.cjs');
 const types = require('twoslash-protocol/types');
diff --git a/dist/index.mjs b/dist/index.mjs
index 8d474686a5930eab6add9525788a5fbf17ce2086..ea8cee5a5b0be18eaa21c51a0b4ebdb108f588ee 100644
--- a/dist/index.mjs
+++ b/dist/index.mjs
@@ -1,4 +1,4 @@
-import ts from 'typescript';
+import ts from 'typescript-legacy';
 import { createTwoslasher as createTwoslasher$1, twoslasher as twoslasher$1 } from './core.mjs';
 export { validateCodeForErrors } from './core.mjs';
 export { T as TwoslashError, d as defaultCompilerOptions, c as defaultHandbookOptions, f as findCutNotations, a as findFlagNotations, b as findQueryMarkers, g as getObjectHash, r as removeTwoslashNotations } from './shared/twoslash.DtmOPsj_.mjs';
