diff --git a/node_modules/@react-email/render/dist/index.js b/node_modules/@react-email/render/dist/index.js
index c9c5838..e025ab6 100644
--- a/node_modules/@react-email/render/dist/index.js
+++ b/node_modules/@react-email/render/dist/index.js
@@ -61,7 +61,7 @@ var import_html_to_text2 = require("html-to-text");
 var import_pretty2 = __toESM(require("pretty"));
 var import_server = require("react-dom/server");
 async function renderToString(children) {
-  const stream = await (0, import_server.renderToReadableStream)(children);
+  const stream = await (0, (import_server.renderToReadableStream || import_server.renderToPipeableStream))(children);
   const html = await readableStreamToString(
     stream
   );
diff --git a/node_modules/@react-email/render/dist/index.mjs b/node_modules/@react-email/render/dist/index.mjs
index 27cc390..059f0ad 100644
--- a/node_modules/@react-email/render/dist/index.mjs
+++ b/node_modules/@react-email/render/dist/index.mjs
@@ -26,9 +26,19 @@ var renderAsPlainText = (component, _options) => {
 // src/renderAsync.ts
 import { convert as convert2 } from "html-to-text";
 import pretty2 from "pretty";
-import { renderToReadableStream, renderToStaticMarkup as renderToStaticMarkup2 } from "react-dom/server";
+
+import react from 'react-dom/server';
+
+const { renderToStaticMarkup, renderToStaticMarkup: renderToStaticMarkup2 } = react;
+const renderToStream =
+  // Note: only available in platforms that support WebStreams
+  // https://react.dev/reference/react-dom/server/renderToString#alternatives
+  react.renderToReadableStream ||
+  // Note: only available in Node
+  react.renderToPipeableStream;
+
 async function renderToString(children) {
-  const stream = await renderToReadableStream(children);
+  const stream = await renderToStream(children);
   const html = await readableStreamToString(
     stream
   );
